app/templates/events/view/sessions.hbs

```diff
@@ -19,16 +19,24 @@
         {{/link-to}}
       {{/tabbed-navigation}}
     </div>
-    <div class="eight wide column">
-      <button class="ui blue button right floated">{{t 'Create Session'}}</button>
-      {{#ui-dropdown class='ui right floated icon top right pointing dropdown button'}}
-        {{t 'Export'}}
-        <i class="download icon"></i>
+    <div class="eight wide {{unless device.isMobile 'right aligned'}} column">
+      {{#ui-dropdown class='compact selection'}}
+        <div class="default text">
+          {{t 'Export As'}}
+        </div>
+        <i class="dropdown icon"></i>
         <div class="menu">
-          <div class="item">{{t 'CSV'}}</div>
-          <div class="item">{{t 'PDF'}}</div>
+          <div class="item" data-value="PDF">
```

Comment:
should I change them on other pages too i.e `events/<event-id>/tickets/orders` and `events/<event-id>/tickets/attendees`